packages/react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts

```diff
@@ -116,12 +99,22 @@ export const getMenuItemStyles = memoizeFunction(
             color: palette.neutralPrimary,
           },
         },
-        ...getItemHighContrastStyles(),
       },
       rootExpanded: {
         backgroundColor: ContextualMenuItemBackgroundSelectedColor,
         color: semanticColors.bodyTextChecked,
-        ...getItemHighContrastStyles(),
+        selectors: {
+          '.ms-ContextualMenu-submenuIcon': {
+            [HighContrastSelector]: {
+              // icons inside of anchor tags are not properly inheriting color in high contrast
```

Comment:
After looking, I think it's because of the `forced-color-adjust: none` on the parent + a separate icon color style coming from another place

Reply:
the oddity is that it only affects anchor tags, not buttons